src/main/java/org/gridsuite/ds/server/service/notification/NotificationService.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.ds.server.service.notification;
+
+import org.springframework.messaging.Message;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public interface NotificationService {
+    void sendMessage(Message<String> message, String bindingName);
```

Comment:
Should not be part of the API as never called by anything except implementation.

Reply:
Yes! Done.